DQM/CastorMonitor/src/CastorDigiMonitor.cc

```diff
@@ -31,12 +53,31 @@ void CastorDigiMonitor::bookHistograms(DQMStore::IBooker& ibooker,
 	const edm::Run& iRun, const edm::EventSetup& iSetup)
 {
   char s[60];
-  if(fVerbosity>0) std::cout << "CastorDigiMonitor::beginRun (start)" << std::endl;
+  string st;
+  if(fVerbosity>0) LogPrint("CastorMonitorModule")<<"Digi bookHist(start)";
+
+  getDbData(iSetup);
+
   char sTileIndex[50];
-  sprintf(sTileIndex,"Tile(=moduleZ*16+sector#phi)");
+  sprintf(sTileIndex,"Cell(=moduleZ*16+sector#phi)");
 
   ievt_=0;
 
+  ibooker.setCurrentFolder(subsystemname_);
+  hBX = ibooker.bookProfile("average E(digi) in BX",
+    "Castor average E (digi);Event.BX;fC", 3601,-0.5,3600.5, 0.,1.e10,"");
+  hBX->getTProfile()->SetOption("hist");
```

Comment:
Please avoid using root objects directly. The option that you are setting will have no effect on the final plot in DQM GUI. In order to set a draw option you either have to use Customise menu in DQM GUI or add render plugin to DQM GUI.